### PR TITLE
Update for SDL v6.1

### PR DESCRIPTION
**Pull request by joeljfischer**
Use menu and choice set managers

### CODE REVIEW

**Comment by NicoleYarroch** [comment]
The link to DarkSky is broken in the README

**Comment by NicoleYarroch** [comment]
The *Supporting Files* folder in the Xcode directory is red.

**Comment by NicoleYarroch** [comment]
Works well. I think the 1000 line SmartDeviceLinkService.m file should be broken into smaller, more readable files.

**Comment by joeljfischer** [comment]
@NicoleYarroch fixed those weirdness things.

I don't disagree about breaking it apart...that'll just take a lot more time. It did use to be like 1600 lines before the managers though so...progress lol. I also added some pragmas to help.

**Comment by NicoleYarroch** [comment]
Should we add the *Pod* folder to the *.gitignore* and add instructions to the README on how to install the SDL library via CocoaPods?

**Comment by joeljfischer** [comment]
🤷🏻‍♂️ We could. I've just been leaving it as is for now.

File: SmartDeviceLinkService.m [hunk 1]
```diff
@@ -104,9 +75,11 @@ - (void)resetProperties {
 
 - (void)start {
     [self resetProperties];
+
+    self.localization = [Localization defaultLocalization];
     
     // Change which config you need based on if you want to connect to a TDK (default) or a wifi based emulator (debug)
-    SDLLifecycleConfiguration *lifecycleConfig = [SDLLifecycleConfiguration defaultConfigurationWithAppName:@"MobileWeather" appId:@"330533107"];
+    SDLLifecycleConfiguration *lifecycleConfig = [SDLLifecycleConfiguration defaultConfigurationWithAppName:@"MobileWeather" fullAppId:@"330533107"];
 //    SDLLifecycleConfiguration *lifecycleConfig = [SDLLifecycleConfiguration debugConfigurationWithAppName:@"MobileWeather" appId:@"330533107" ipAddress:@"192.168.1.61" port:2776];
```
Comment (NicoleYarroch) [inline SmartDeviceLinkService.m]: The debug configuration is deprecated

**Comment by NicoleYarroch** [comment]
Have you tested the app with Manticore? The primary graphic images are barely visible due to the image color being almost identical to the background color. It it possible to template the primary graphic image so it looks good when Manticore is updated to Core 5.0?

File: SmartDeviceLinkService.m [hunk 2]
```diff
@@ -342,7 +342,7 @@ - (void)sendWeatherConditions:(WeatherConditions *)conditions withSpeak:(BOOL)wi
         self.manager.screenManager.textField3 = [conditions.precipitation stringValueForUnit:percentageType shortened:YES localization:self.localization];
         self.manager.screenManager.textField4 = [conditions.windSpeed stringValueForUnit:speedType shortened:YES localization:self.localization];
 
-        self.manager.screenManager.primaryGraphic = [SDLArtwork artworkWithImage:[[ImageProcessor sharedProcessor] imageFromConditionImage:conditions.conditionIcon] name:conditions.conditionIcon asImageFormat:SDLArtworkImageFormatPNG];
+        self.manager.screenManager.primaryGraphic = [SDLArtwork artworkWithImage:[[[ImageProcessor sharedProcessor] imageFromConditionImage:conditions.conditionIcon] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate] name:conditions.conditionIcon asImageFormat:SDLArtworkImageFormatPNG];
```
Comment (NicoleYarroch) [inline SmartDeviceLinkService.m:345]: There is another section of the code on L444 where the primary graphic is also set. This image isn't templated.

Comment (NicoleYarroch) [reply SmartDeviceLinkService.m:345]: The menu icons should be templated too.

**Review by NicoleYarroch — APPROVED** [review]

**Pull request merged**